packages/plugin-devtools/test/devtools.test.ts

```diff
@@ -0,0 +1,735 @@
+        root.innerHTML = '<h1>Title</h1><p>a<b>b</b>c</p><p>def<br>gh<i>i</i></p><div>div</div>';
+        wrapper.appendChild(root);
+
+        Object.keys(localStorage).forEach(key => {
+            if (key.indexOf('OwlUI') !== -1) {
+                localStorage.removeItem(key);
+            }
+        });
+
+        editor = new JWEditor(root);
+        editor.loadConfig({
+            debug: true,
+        });
+
+        await editor.start();
+
+        devtools = editor.el.querySelector('jw-devtools');
+    });
+    after(async () => {
+        editor.stop();
+        document.body.removeChild(wrapper);
+    });
+
+    describe('DevToolsComponent', () => {
+        it('should display the devtools', async () => {
+            expect(!!devtools).to.equal(true);
+        });
+        it('should display the devtools closed as bottom bar', async () => {
+            expect(devtools.classList.contains('closed')).to.equal(true);
+        });
+        it('should open the devTools', async () => {
+            const navbar = devtools.querySelector('devtools-navbar');
+            navbar.dispatchEvent(new MouseEvent('mousedown', { bubbles: true }));
+            await nextTick();
+            navbar.dispatchEvent(new MouseEvent('mouseup', { bubbles: true }));
+            await nextTick();
+            navbar.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+            await nextTick();
+            expect(devtools.classList.contains('closed')).to.equal(false);
+        });
+        it('should display the devTools with "Inspector" by default when open', async () => {
+            const button = devtools.querySelector('devtools-navbar button');
+            expect(button.textContent).to.equal('Inspector');
+            expect(button.classList.contains('selected')).to.equal(true, 'button is selected');
+            const panel = devtools.querySelector('devtools-panel.inspector.active');
+            expect(!!panel).to.equal(true, 'panel is active');
+        });
+        it('should resize the devTools', async () => {
+            const navbar = devtools.querySelector('devtools-navbar');
+            const pos = navbar.getBoundingClientRect();
+            const begin = { clientX: pos.left + 5, clientY: pos.top + 5, bubbles: true };
+            const step = { clientX: pos.left + 15, clientY: pos.top + 25, bubbles: true };
+            const end = { clientX: pos.left + 30, clientY: pos.top - 50, bubbles: true };
+            navbar.dispatchEvent(new MouseEvent('mousedown', begin));
+            await longNextTick();
+            window.dispatchEvent(new MouseEvent('mousemove', step));
+            await longNextTick();
+            window.dispatchEvent(new MouseEvent('mousemove', end));
+            await longNextTick();
+            navbar.dispatchEvent(new MouseEvent('mouseup', end));
+            await nextTick();
+            navbar.dispatchEvent(new MouseEvent('click', end));
+            await nextTick();
```

Comment:
Maybe, just maybe, we might need a testUtils.dragAndDrop function. It wouldn't be of great use now but it might be if we have more dragging stuff coming later.